client/src/game/saga.js

```diff
@@ -0,0 +1,70 @@
+import { all, put, select, takeLatest, call } from 'redux-saga/effects';
+import axios from 'axios';
+import { selectCurrentUser } from '../auth/selectors';
+import { selectCurrentGameID } from './selectors';
+import { actions as gameActions } from './slice';
+import { actions as alertActions } from '../app/containers/Alert/slice';
+
+export function* handleExitGame({ payload: { history } }) {
+  yield call(history.push, '/home');
```

Comment:
This doesn't necessarily need to be yielded but it won't break anything if you do

Reply:
Basically you're not actually yielding anything.